packages/backend/src/league/league_controller.ts

```diff
@@ -266,4 +317,23 @@ export class LeaguesController extends ControllerWrap<League> {
       }),
     );
   }
+
+  private async checkAndCompressImage<T>(
```

Comment:
Isn't this already a function in the user controller?

Reply:
Yea, I mean it *could* be refactored out into a static function -- it involves doing more work on the callee side though since it has reference to `this`.

Reply:
eh that's fine